Gemfile

```diff
@@ -42,7 +42,7 @@ group :test do
 end
 
 group :acceptance do
-  gem 'beaker', '~> 2.2'
+  gem "beaker", :git => 'git://github.com/puppetlabs/beaker.git', :ref => '7d0cccf22cc7289113de862fe4a4494872f574bb'
```

Comment:
Is this affected by https://github.com/puppetlabs/puppetdb/pull/1317 which was just merged in?